.github/workflows/platform-exec-sanity-tests.yml

```diff
@@ -0,0 +1,72 @@
+name: Platform Executable Sanity testing
+on:
+  workflow_dispatch:
+  workflow_run:
+    # wait for Platform Executable Release to complete
+    workflows: ["Platform Executable Release"]
```

Comment:
This workflow will be executed after `Platform Executable Release` irrespective of `draft-release` on the `main` branch.  Are we adding the condition somewhere to check it should be executed in `draft-release`?

Reply:
Yes, Since this testing checks if the executables are installable and tests with a basic command, thought it would be okay to execute it always since executables are created from scratch, in case of actual release too. Let me know if you feel we should only trigger it once.